plugins/importexport/templates/resultsExport.tpl

```diff
@@ -0,0 +1,76 @@
+{**
+ * plugins/importexport/native/templates/results.tpl
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
+ * Copyright (c) 2003-2020 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * Result of operations this plugin performed
+ *}
+{if $errorsFound}
+	{translate key="plugins.importexport.native.processFailed"}
+{else}
+	{translate key="plugins.importexport.native.export.completed"}
+	{translate key="plugins.importexport.native.export.completed.downloadFile"}
+
+	<div id="exportIssues-tab">
+		<script type="text/javascript">
+			$(function() {ldelim}
+				// Attach the form handler.
+				$('#exportIssuesXmlForm').pkpHandler('$.pkp.controllers.form.FormHandler');
+			{rdelim});
+		</script>
+		<form id="exportIssuesXmlForm" class="pkp_form" action="{plugin_url path="downloadExportFile"}" method="post">
+			{csrf}
+			<input type="hidden" name="downloadFilePath" id="downloadFilePath" value="{$exportPath}" />
+			{fbvFormArea id="issuesXmlForm"}
+				{fbvFormButtons submitText="plugins.importexport.native.export.download.results" hideCancel="true"}
+			{/fbvFormArea}
+		</form>
+	</div>
+{/if}
+
+{if array_key_exists('warnings', $errorsAndWarnings) && $errorsAndWarnings.warnings|@count > 0}
+	<h2>{translate key="plugins.importexport.common.warningsEncountered"}</h2>
+	{foreach from=$errorsAndWarnings.warnings item=allRelatedTypes key=relatedTypeName}
+		{foreach from=$allRelatedTypes item=thisTypeIds key=thisTypeId}
+			{if $thisTypeIds|@count > 0}
+				<p>{$relatedTypeName} {if $thisTypeId > 0} (Id: {$thisTypeId}) {/if}</p>
```

Comment:
Unless it's really not required, the `|escape` modifier should be used when variables are included in Smarty templates.
Also, `Id:` should be a locale string (with the ID specified as a variable, i.e.: `Id: {$id}`)

Reply:
Fixed